languages/java/oso/src/main/java/com/osohq/oso/Host.java

```diff
@@ -170,37 +172,38 @@ public boolean isa(long instanceId, String classTag)
      * Convert Java Objects to Polar (JSON) terms.
      *
      * @param value Java Object to be converted to Polar.
-     * @return JSONObject Polar term of form: {@code {"id": _, "offset": _, "value": _}}.
+     * @return JSONObject Polar term of form: {@code {"id": _, "offset": _, "value":
+     *         _}}.
      */
     public JSONObject toPolarTerm(Object value) throws Exceptions.OsoException {
         // Build Polar value
         JSONObject jVal = new JSONObject();
-        if (value.getClass() == Boolean.class) {
+        if (value != null && value.getClass() == Boolean.class) {
```

Comment:
How about handling the null check first?

Reply:
yeah, I just wanted to use the same code as the external instance, so I'd have to duplicate that at the top. I thought this was slightly clearer but can change.

Reply:
we can do this as a follow-up if needed, merging now